docs/user-guide/configure-certificates.md

```diff
@@ -1,30 +1,130 @@
+
+* If you are using a USS keystore, then the script zowe-setup-certificates.env is the configuration step required to create the USS directory that contains the certificate. This is described in detail in Configuring Zowe certificates in a USS KeyStore.
+
+* If you are using a key ring, the sample JCL member `ZWEKRING` provided in the PDS library `SZWESAMP` contains the security commands to create a key ring and manage its associated certificates. This is described in Configuring Zowe certificates in a key ring which provides instructions for how to configure Zowe to work with the following certificates.
+    * a self-signed certificate
+    * a certificate signed with an existing certificate authority
+    * an existing certificate already held in the SAF database that can be added to the Zowe key ring.
+
+For both scenarios, where the certificate is held in a USS Java Keystore or a z/OS key ring, the `USS KEYSTORE_DIRECTORY` is still required which is created with the script zowe-setup-certificates.sh.
+
+* In the USS scenario, this directory holds the `.cer` and `.pem` files for the certificate itself.
+* In the key ring scenario, this directory stores the location and name of the Zowe key ring and its certificates.
+
+### Enable certificate validation without hostname validation
+
+To enable certificate validation without hostname validation, set `zowe.verifyCertificates: NONSTRICT`. Using this setting, the certificate Common Name or Subject Alternate Name (SAN) is not checked. Disabling this parameter facilitates deployment to environments where certificates are valid but do not contain a valid hostname. This configuration is for development purposes only and should not be used for production.
+
+A proper setup of the truststore is mandatory to successfully start Zowe with `zowe.verifyCertificates: STRICT`.
+
+## Choosing USS Keystore or z/OS Key ring to store Zowe certificates
+
+Zowe supports certificates that are stored either in a USS directory **Java KeyStore** in the `.p12` format or, alternatively, certificates held in a **z/OS Key ring**. Use of a z/OS keystore is the recommended option for storing certificates if system programmers are already familiar with the certificate operation and usage.
+Creating a key ring and connecting the certificate key pair requires elevated permissions. When the TSO user ID does not have the authority to manipulate key rings and users want to create a Zowe sandbox environment or for testing purposes, the USS keystore is a good alternative.
+
+### Create a certificate authority and use it to self-signed a certificate
```

Comment:
To better follow our style guide: Creating a certificate authority and use it ...